docs/source/protocol.rst

```diff
@@ -30,28 +30,29 @@ Network Send Number
 - Begins at 0 on each client and goes up by 1 each time we attempt a metrics network upload.
 - Does not go up on network retries.
 
-Relative Timestamp
-++++++++++++++++++
+Image
++++++
 
-- 64 bit signed integer
-- GVariant symbol: ``x``
-- See: http://linux.die.net/man/3/clock_gettime
+- String
+- GVariant symbol: ``s``
 
-Absolute Timestamp
-++++++++++++++++++
+Site
+++++
 
-- 64 bit signed integer
-- GVariant symbol: ``x``
-- Nanoseconds since the Unix epoch
-- See: http://linux.die.net/man/3/clock_gettime
+- String
+- GVariant symbol: ``a{ss}``
 
-Machine ID
-++++++++++
+Dualboot
+++++++++
 
-- Array of 16 unsigned bytes
-- GVariant symbol: ``ay``
-- Should uniquely identify a user's machine.
-- See: http://man7.org/linux/man-pages/man5/machine-id.5.html
+- Boolean
+- GVariant symbol: ``b``
+
+Live
+++++
+
+- Boolean
+- GVariant symbol: ``b``
 
 Singular Metrics
```

Comment:
This section needs update for the new type of each singular metric

Reply:
I’m not sure to understand this comment. Do we need something more than an updated GVariant symbol?

Reply:
No, that's all!